src/cms/preview-templates/ColorCollectionPreview.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ColorList from '../../components/Color/ColorList'
+
+const ColorCollectionPreview = ({ entry, widgetsFor }) => {
+    return (
+        <section>
+            <div
+                css={{
+                    borderBottom: 3,
+                    borderBottomStyle: 'double',
+                    marginBottom: 30
+                }}
+            >
+                <h1>{entry.getIn(['data', 'title'])}</h1>
+                <h2>Version: {entry.getIn(['data', 'version'])}</h2>
+            </div>
+            <div>
+                { widgetsFor('colorGroups').map((colorGroups, index) => 
+                <ColorList 
+                    key={index}
+                    name={colorGroups.getIn(['data', 'name'])}
+                    colors={colorGroups.getIn(['data', 'colors'])} // Returns an 'array of objects' containing color 'data' from CMS
```

Comment:
`colorGroups.toJS().data.colors` will return array you want to pass into colorList